method/link.js

```diff
@@ -89,26 +98,19 @@ module.exports = function Link (ssb) {
     findGroupByFeedId (feedId, cb) {
       if (!isFeed(feedId)) return cb(new Error('requires a valid feedId'))
 
-      const query = [{
-        $filter: {
-          value: {
-            author: feedId, // link published by this person
-            content: {
-              type: 'link/feed-group',
-              parent: feedId, // and linking from that feedId
-              tangles: {
-                link: { root: null, previous: null }
-              }
-            }
-          }
-        }
-      }]
-
       pull(
-        // NOTE: using ssb-query instead of ssb-backlinks
-        // because the backlinks query will get EVERY message which contains the groupId in it, which will be a LOT for a group
-        // then filters that massive amount down to the ones which have the dest in the right place
-        ssb.query.read({ query }),
+        ssb.db.query(
+          where(
+            and(
+              author(feedId),
+              dbType('link/feed-group'),
+              slowEqual('value.content.parent', feedId),
+              slowEqual('value.content.tangles.link.root', null),
+              slowEqual('value.content.tangles.link.previous', null),
```

Comment:
can revisit once we have this all working. We may not want indexes for these fields, I'm not sure

Reply:
was gonna optimize this but then noticed that i couldn't find `findGroupByFeedId` being used in any repos actually. should i just remove it instead?

Reply:
Used in `@ssb-graphql/main`

Reply:
switched to `equal` now